lib/appsignal/hooks/http.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "http/client"
+
+module Appsignal
+  class Hooks
+    # @api private
+    class HttpHook < Appsignal::Hooks::Hook
+      register :http_rb
+
+      def dependencies_present?
+        Appsignal.config && Appsignal.config[:instrument_http_rb]
```

Comment:
```suggestion
        defined?(HTTP::Client) && Appsignal.config && Appsignal.config[:instrument_http_rb]
```

Let's check if the class we want to extend is present.